src/docs/patterns/tags.mdx

```diff
@@ -0,0 +1,230 @@
+
+Always double-check the color choices to ensure that the tag meets the [accessibility requirements](/design/accessibility).
+
+<ImageGrid
+    images={[YellowTagsDont, YellowTags]}
+    captionsUnderImages={[
+        `DONT: Do not use color combinations that make tags hard to read. This is especially important for light colors such as yellow and orange.`,
+        `DO: Always select colors that make the text readable.`,
+    ]}
+/>
+
+# Labels
+
+To ensure users can easily read the tag while keeping the tag size as small as possible, the font style of tags is 10dp, all caps, and bolded. Do not use lowercase letters on tags, as they are less legible at a small font size.
+
+<ImageGrid images={[LowerCase]} captionsUnderImages={[`DONT: Do not use lowercase letters in tags.`]} />
+
+## Text-Only
+
+Use only texts inside a tag. Do not use additional elements such as leading icons and trailing remove icons.
+
+<ImageGrid
+    images={[TextOnly]}
+    captionsUnderImages={[
+        `DONT: Do not include anything in tags other than texts. Chips should be used in this situation.`,
+    ]}
+/>
+
+## Short Labels
+
+Keep the label short. Ideally, the tag label should contain only 1-2 words that form an adjective.
+
+<ImageGrid
+    images={[LongWords]}
+    captionsUnderImages={[
+        `AVOID: Avoid using more words than necessary. "Updates available" is shorter and can convey the same message.`,
+    ]}
+/>
+
+Be careful when designing tags with a fixed width, as the tag label can overflow when translated into a new language.
+
+<ImageGrid images={[I18N]} captionsUnderImages={[`CAUTION: Fixed-width tags may cause labels to overflow.`]} />
+
+# Multiple Tags
+
+List items can be flagged with more than one tag. Avoid assigning different tags with the same or similar variants and colors.
+
+<ImageGrid
+    images={[MultiTagSameColor]}
+    captionsUnderImages={[
+        `AVOID: Avoid using the same variant and color for different tags, as it becomes difficult to distinguish tags from a glance.`,
+    ]}
+/>
+
+Order tags by frequency to minimize the chance of tags shifting horizontally in a list.
+
+<ImageGrid
+    images={[MultiTagShiftingDont, MultiTagShiftingDo]}
+    captionsUnderImages={[
+        `AVOID: Avoid "tag shifting" in lists.`,
+        `DO: Minimize users' effort to scan through tags by placing the most frequently used tag to the right.`,
+    ]}
+/>
+
+If you are running out of horizontal spaces, such as when designing for mobile devices, there are a few ways to reposition tags to a more appropriate place.
+
+<ImageGrid
+    images={[MultiTagInfoLine, MultiTagEllipsis, MultiTagWrap, MultiTagWrapInline]}
+    captionsUnderImages={[
+        `Tags can be placed next to list item texts.`,
+        `AVOID: Avoid using ellipsis in tags. However, if an ellipsis is unavoidable, show a tooltip with the full label.`,
+        `Tags can be wrapped to a new line as "inline blocks".`,
+        `DONT: Never wrap within a tag.`,
```

Comment:
Wouldn't wrapping within a tag look like this:
<img width="147" alt="image" src="https://github.com/etn-ccis/blui-doc-it/assets/29152776/3462c92f-407b-4851-9f19-a6bc4fe6f4af">

Is that so bad?

Reply:
The default tag wrap usually looks like what I prototyped, but I like yours better